src/CFixer/main.cpp

```diff
@@ -5,6 +5,19 @@
 //# of this license document, but changing it is not allowed.
 //#
 
+#include <CopyrighFixer/bar.h>
+#include <QString>
+#include <QDebug>
```

Comment:
use please std cout object

Reply:
remove not used include  
```
#include <QDebug>
```
And remove QString because you alredy include them in the <CopyrighFixer/bar.h>
```
#include <QString>
```

Reply:
Done!

Reply:
Its not "Done" you need fix this before close